### DISCUSSION TITLE
export_openMVS

### DISCUSSION THREAD

**Discussion by AdrianKriger** [cdcseacave/openMVS#698] [Q&A] [answered]
This is a wonderful library.
The dense reconstruction is spectacular.
Thank you.

I want to geo-reference the project (with ground control points). I cannot build the `ui_openMVG_control_points_registration` from [openMVG](https://github.com/openMVG/openMVG) that feeds into your pipeline successfully.
To get around this; it's possible to do it after with [SfM_georef](https://www.lancaster.ac.uk/staff/jamesm/software/sfm_georef.htm) or [VisualSFM](http://ccwu.me/vsfm/index.html)

I need to export the openMVS (scene_dense.mvs) project file. 
Does `InterfaceVisualSFM` support this? - like: `InterfaceVisualSFM scene_dense.mvs`.

**Comment by cdcseacave** [comment, marked as answer]
the geo-reference absolute position can be usually represented as a trasform from a local to an absolute coordinate space, ex from ENU to ECEF; all you need to do is reconstruct the SfM scene in ENU, compute the transoform to ECEF (OpenMVG should be able to do this), proceed with OpenMVS as usual in the ENU coordinate space, and at the end transform the mesh or the point cloud (whjatever you need) to ECEF, etc.

please do not use VisualSFM, it is deprecated and not performing as well

**Reply by AdrianKriger** [reply]
You are very kind to respond so promptly.

`openMVG_main_geodesy_registration_to_gps_position` can adjust a scene coordinate system to fit a GPS datum (typically from the photo centers). 
What I need is `ui_openMVG_control_points_registration` to transform a scene with ground control (GCP). 
I can't build the tool successfully and was hoping to export the `openMVS` `dense_scene.mvs` and transform with another library. Either [SfM_georef](https://www.lancaster.ac.uk/staff/jamesm/software/sfm_georef.htm) or [VisualSFM](http://ccwu.me/vsfm/).

Can I export the `dense_scene.mvs` to another format?

**Reply by cdcseacave** [reply]
each openMVS step exports the computed data both in MVS and PLY format

**Reply by AdrianKriger** [reply]
Thank you.
